docs/concepts/llm_convenience_wrappers.md

```diff
@@ -18,14 +18,14 @@ You can initialize an [`OpenAIChat`](../api/chat/models.md#mirascope.chat.models
 ```python
 from mirascope import OpenAIChat, Prompt
 
-class RecipePrompt:
-		"""
-		Recommend recipes that use {ingredient} as an ingredient
-		"""
+class RecipePrompt(Prompt):
+	"""
```

Comment:
this still seems like it's indented one too many compared to e.g. ChefPrompt?

Reply:
see above comment regarding github